slackroll

```diff
@@ -1609,6 +1670,24 @@ def analyze_changes(local_list, remote_list, persistent_list): # XXX THIS FUNCTI
     # Sync database to disk
     persistent_list.sync()
 
+def verify_operation_and_args(operation, op_num_args, args): # Verify number of arguments
```

Comment:
I'd move the op_num_args argument to be the first or the last one, so as to keep "operation" and "args" next to each other. I made a mistake when choosing the argument order in verify_num_args(), which you probably took as a reference. That could be changed too.

Reply:
Ok, I've updated the signatures and usages of both of those methods.